roles/variables/defaults/main.yml

```diff
@@ -69,6 +69,9 @@ kerberos_kafka_broker_primary: "{{ (hostvars[ groups['kafka_broker'][0] | defaul
 
 open_file_limit: 500000
 
+### Variable to define minimum free disk space ratio for installation.
```

Comment:
The documentation/comment and the variable name are clashing in my mind. The variable is a maximum used disk space, while the documentation mentions free space. I'd suggest naming it something like `max_used_disk_ratio` to make it easier to grasp what is it that is being defined

Reply:
I disagree.  To me, the way you describe it is more confusing. The variable is the ratio of free space that is required, which is what the documentation clearly states.

Reply:
You're right, I have misread the code, thanks. 

I would still suggest a more descriptive name like `free_disk_space_ratio` or similar, as the name `disk_ratio` in the inventory tells very little what it is about (if overridden in the inventory, ofc)

Reply:
That's fair, let me update that.